nncf/experimental/torch2/model_transformer.py

```diff
@@ -34,8 +38,9 @@ class PT2ModelTransformer(ModelTransformer[GraphModelWrapper]):
     def __init__(self, model: GraphModelWrapper):
         super().__init__(model)
 
-        self._command_transformation_ordered_pairs = [
-            (PT2InsertionCommand, self._apply_insertion_transformation),
+        self._command_transformation_ordered_pairs: TRANSFORMATION_PAIRS = [
+            (PT2InsertionCommand, self._apply_insertion_transformations),
+            (PTBiasCorrectionCommand, self._apply_bias_correction_transformations),
         ]
```

Comment:
Just for a bit of saferty
```suggestion
        self._command_transformation_ordered_pairs: TRANSFORMATION_PAIRS = (
            (PT2InsertionCommand, self._apply_insertion_transformations),
            (PTBiasCorrectionCommand, self._apply_bias_correction_transformations),
        )
```

Reply:
done